src/azure-cli/azure/cli/command_modules/iot/_help.py

```diff
@@ -402,6 +402,9 @@
   - name: Create an IoT Hub with the standard pricing tier S1 and 4 partitions, in the 'westus' region, with tags.
     text: >
         az iot hub create --resource-group MyResourceGroup --name MyIotHub --location westus --tags a=b c=d
+  - name: Create an IoT Hub with a system-assigned managed identity, and assign a role and scope to a storage account for the created identity.
+    text: >
+        az iot hub create --resource-group MyResourceGroup --name MyIotHub --location westus --assign-identity [system] --role "Storage Blob Data Contributor" --scopes {resourceId}
```

Comment:
@c-ryan-k Hi, our team has a meeting today to discuss how to redesign the parameters of managed identity, the parameters related to managed identity need to be modified to a new interface：
We recommend using `--mi-system-assigned` to enable system-assigned identity and `--mi-user-assigned` with space separated resource IDs to add user-assigned identities instead of a mixed parameter.

For more details, refer to this guideline [doc link](https://github.com/Azure/azure-cli/pull/17473/files), could you please take a look~

Reply:
@c-ryan-k Hi, our team has a meeting today to discuss how to redesign the parameters of managed identity, the parameters related to managed identity need to be modified to a new interface：
We recommend using `--mi-system-assigned` to enable system-assigned identity and `--mi-user-assigned` with space separated resource IDs to add user-assigned identities instead of a mixed parameter.

For more details, please refer to this guideline [doc link](https://github.com/Azure/azure-cli/pull/17473/files), could you please take a look~